sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -2923,10 +3048,14 @@ SUBROUTINE MDLFLD
             DO I=1,IM
                GRID1(I,J)=0.0
                DO L=1,NINT(LMH(I,J))
+               IF(QQR(I,J,L)<spval.and.QQS(I,J,L)<spval.and.QQG(I,J,L)<spval)THEN
```

Comment:
add checking for ZONT, PMID, T, and Q too

Reply:
will do

Reply:
done